### DISCUSSION TITLE
Can't Upgrade to 1.17.0

### DISCUSSION THREAD

**Discussion by akong77** [modoboa/modoboa#2330] [Upgrade]
I want upgrade from 1.14 to 1.17.I has setup python to 3.5.3.

root@mail:~# python --version
Python 3.5.3

But when I upgrade.It's will show this error message.

`(env) root@mail:~# pip install modoboa==1.17.0
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
ERROR: Could not find a version that satisfies the requirement modoboa==1.17.0 (from versions: 0.7.0, 1.2.0rc2, 1.2.0, 1.2.1, 1.2.2, 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.3.4, 1.3.5, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.4.5, 1.5.0, 1.5.1, 1.5.2, 1.5.3, 1.6.0, 1.6.1, 1.6.2, 1.6.3, 1.7.0, 1.7.1, 1.7.2, 1.7.3, 1.7.4, 1.8.0, 1.8.1, 1.8.2, 1.8.3, 1.9.0, 1.9.1, 1.10.0, 1.10.1, 1.10.2, 1.10.3, 1.10.4, 1.10.5, 1.10.6, 1.10.7, 1.11.0, 1.11.1, 1.12.0, 1.12.1, 1.12.2, 1.13.0, 1.13.1, 1.14.0)
ERROR: No matching distribution found for modoboa==1.17.0`

How to fix it?

**Comment by Toniob** [comment]
Hi. Did you upgrade your python version in your virtualenv ?
